modules/backend/widgets/Filter.php

```diff
@@ -264,6 +264,14 @@ public function onFilterUpdate()
                 $value = post('options.value.' . $scope->scopeName) ?: null;
                 $this->setScopeValue($scope, $value);
                 break;
+
+            case 'clear':
+                foreach ($this->getScopes() as $scope) {
+                    $this->setScopeValue($scope, null);
+                }
+
+                return ['.control-filter' => $this->render()];
```

Comment:
@robinbonnes Would you mind putting the array key and value on a new line - it makes it more readable as an array.
```suggestion
                return [
                    '.control-filter' => $this->render()
                ];
```

Reply:
@bennothommo personally I don't mind the oneliner array in some cases, especially when there is only one element. Thoughts?

Reply:
@LukeTowers It's subjective, but I justify it in a couple of ways:

- Although it's easier to tell, at a glance, the above as being an array with the `=>` symbol - it's harder when you have two parameters instead, ie. `return ['.control-filter', $this->render()];`
- We already require arrays to split over multiple lines when there's more than one item, so this is just consistent with that.
- It's similar formatting to method signatures that span longer lines, ie:

```
public function someMethod(
    $someReallyLongAttribute = 'A really long default parameter'
) {
```

Reply:
@bennothommo counterpoint to your points :)
> Although it's easier to tell, at a glance, the above as being an array with the => symbol - it's harder when you have two parameters instead, ie. return ['.control-filter', $this->render()];

If it's two elements, it should be two lines, no matter how long it is.

> We already require arrays to split over multiple lines when there's more than one item, so this is just consistent with that.

Not quite, since this is technically one item, with a key provided.

> It's similar formatting to method signatures that span longer lines

I sincerely hope that example does not exist in October anywhere; I do not ever want to see that added in the future. If something is that long of a method parameter / default value, you're doing something wrong with naming / defaults assignment.

Reply:
@LukeTowers they're all fair points (as for point 3, I haven't seen any yet, but rest assured I will deal with that if I ever saw it too).

I'll probably still do it the way I've suggested, because I do find it spaces out the definitions a bit more clearer, but I won't ping anyone if they have implemented it as a one-liner.